tests/cypress.config.ts

```diff
@@ -0,0 +1,26 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+    chromeWebSecurity: false,
+    failOnStatusCode: false,
+    defaultCommandTimeout: 30000,
+    videoUploadOnPasses: false,
+    reporter: 'cypress-multi-reporters',
+    reporterOptions: {
```

Comment:
I see in the test report here https://github.com/Jahia/personal-api-tokens/pull/146/checks?check_run_id=47997319797 that the name of the xml file are built with a random name like [results-4332f6d81419ddba35c3acc04bf85e0b.xml](https://github.com/Jahia/personal-api-tokens/runs/47997319797#r0)
Is it possible to have the name of the suite as the file name instead, like it's done for jExperience for example